rust/examples/src/bin/place_order.rs

```diff
@@ -0,0 +1,184 @@
+use std::str::FromStr;
+
+use ellipsis_client::EllipsisClient;
+use phoenix::state::Side;
+use phoenix_sdk::sdk_client::SDKClient;
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::commitment_config::CommitmentConfig;
+#[allow(unused_imports)]
+use solana_sdk::{
+    instruction::Instruction,
+    program_pack::Pack,
+    pubkey::Pubkey,
+    signature::{read_keypair_file, Keypair},
+    signer::Signer,
+};
+use spl_token::state::Mint;
+#[tokio::main]
+async fn main() {
+    // Connect to the solana network and get the market address
+    let network_url = "https://api.devnet.solana.com";
+    let market = Pubkey::from_str("CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N").unwrap();
+
+    let trader = Keypair::new();
+    let rpc_client =
+        RpcClient::new_with_commitment(network_url.to_string(), CommitmentConfig::confirmed());
+    let client = EllipsisClient::from_rpc(rpc_client, &trader).unwrap();
+    println!("Trader pubkey: {}", trader.pubkey());
+    let sdk = SDKClient::new_from_ellipsis_client_with_all_markets(client)
+        .await
+        .unwrap();
+    // Alternatively, read from keypair file for the payer
+    // let path = "~/.config/solana/id.json";
+    // let auth_payer = read_keypair_file(&*shellexpand::tilde(path)).unwrap();
+
+    println!("trader {}", sdk.trader);
+
+    // Only relevant for devnet
+    let instructions = create_airdrop_spl_ixs(&sdk, &market, &trader.pubkey())
+        .await
+        .unwrap();
+    sdk.client
+        .request_airdrop(&trader.pubkey(), 1_500_000_000)
```

Comment:
why do we need request_airdrop on top of airdrop_spl_ixs?

Reply:
nit: prefer a whole number of SOL, e.g. 1b or 2b lamports. Does this hit the solana-airdrop directly or does it go through our token faucet?

Reply:
Yep, it requests devnet SOL, like in solana airdrop, to be used for sending the transaction. (Does not have to do with our token faucet)